puppetboard/app.py

```diff
@@ -192,8 +193,12 @@ def index(env):
         metrics=metrics,
         nodes=nodes_overview,
         stats=stats,
+<<<<<<< HEAD
+        refreshrate=refreshrate
+=======
```

Comment:
please remove this line (this is a diff between old and new, be sure to only merge the needed params)